manifests/kubernetes-mcp-server-clusterrole-admin.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubernetes-mcp-server-admin
+  labels:
+    app.kubernetes.io/name: kubernetes-mcp-server
+    app.kubernetes.io/version: v0.0.51
+rules:
+- apiGroups: ["*"]
```

Comment:
Since we rely on OAuth and use the token's privileges, I think MCP Server should have basic privileges (only TokenReview maybe?). Otherwise, how do we know that mcp server's service account is used or token's privileges is used?.